src/main/java/base/CsvDataProviders.java

```diff
@@ -0,0 +1,42 @@
+package base;
+
+import com.opencsv.CSVReader;
+import org.testng.annotations.DataProvider;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileReader;
+import java.io.IOException;
+import java.lang.reflect.Method;
+import java.util.*;
+
```

Comment:
suggestion: you can leave some javadocs to describe what is the scope of this class and usage of each method (only one for now :) )

Reply:
Will be resolved in next PR

Reply:
you mean, in this PR :) thank you!